jdaviz/app.vue

```diff
@@ -188,6 +189,7 @@ export default {
   data() {
     return {
       outputCellHasHeight: false,
+      show: true,
```

Comment:
does this only apply to the golden-layout section (not the entire app)?  If so, maybe we should be more clear in the naming to avoid it getting used somewhere else?